suite/runsuite_common_pre.cmake

```diff
@@ -347,9 +347,9 @@ elseif (arg_use_make)
   if (have_cygwin)
     # seeing errors building in parallel: pdb collision?
     # can't repro w/ VERBOSE=1
-    set(CTEST_BUILD_COMMAND_BASE "${MAKE_COMMAND} -j2")
+    set(CTEST_BUILD_COMMAND_BASE "${MAKE_COMMAND} -j2 VERBOSE=1")
```

Comment:
I'm seeing multiple changes that presumably should be temporary but are not marked that way!  Is this ready for review?  Please look through it.

Reply:
Do not review at this time, wait for PTAL (debugging clang error).